### PR TITLE
Reprint error

### PR DESCRIPTION
**Pull request by goretkin**
Attempt at https://github.com/timholy/Revise.jl/issues/407

### CODE REVIEW

**Comment by timholy** [comment]
Ideally it might also have a docstring and a brief mention in the docs.

**Comment by goretkin** [comment]
Thanks for the feedback. Please let me know if `Revise.errors()` looks better or if the doc changes can be improved.

I wasn't sure if I should add a new line to `NEWS.md`.

File: test/runtests.jl [hunk 1]
```diff
@@ -1504,23 +1504,39 @@ end
...
 
+        # test errors are not re-reported
+        logs, _ = Test.collect_test_logs() do
+            yry()
+        end
+        @test length(logs) == 1
+
+        # test error re-reporting
+        logs,_ = Test.collect_test_logs() do
+            Revise.report_errors()
```
Comment (timholy) [inline test/runtests.jl]:
```suggestion
            Revise.errors()
```

File: NEWS.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 This file describes only major changes, and does not include bug fixes,
 cleanups, or minor enhancements.
 
+## Revise master
```
Comment (timholy) [inline NEWS.md]:
```suggestion
## Revise 2.5
```

File: docs/src/user_reference.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # User reference
 
 There are really only four functions that a user would be expected to call manually:
```
Comment (timholy) [inline docs/src/user_reference.md]:
```suggestion
There are really only five functions that a user would be expected to call manually:
```

**Comment by goretkin** [comment]
Thanks for catching those things!

**Comment by timholy** [comment]
Thanks for an excellent contribution! Tagging 2.5 now.

**Pull request merged**